### PR TITLE
update sqoop-hive sharelib for HDP 2.6.3

### LINKED ISSUE #1112: update sqoop-oozie workarounds for HDP 2.6.3
So that oozie+sqoop still works

Update this abomination: https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/recipes/oozie.rb#L229-L252

### PR DESCRIPTION
**Pull request by aespinosa**
Insert the workaround hacks after sharelibupdate

Otherwise it is writing the workarounds in the *previous* sharelib
version.  I don't know why this was working when I first wrote this
`¯\_(ツ)_/¯`.

Fixes #1112 

Added `to-backport` tag for 3.3 once it gets released

### CODE REVIEW

**Comment by aespinosa** [comment]
discussed with @vt0r and decided to get this merged and just issue a new PR for the hotfix workaround.

File: cookbooks/bcpc-hadoop/recipes/oozie.rb [hunk 1]
```diff
@@ -226,14 +226,25 @@
...
-                 'workaround for 2.6.1]', :immediately
+                 'workaround for 2.6.3]', :immediately
 end
 
-ruby_block 'oozie sharelib sqoop-action workaround for 2.6.1' do
+ruby_block 'oozie sharelib sqoop-action workaround for 2.6.3' do
   block do
     active_release = node['bcpc']['hadoop']['distribution']['active_release']
-    if active_release == '2.6.1.17-1'
+    if active_release == '2.6.3.0-235'
```
Comment (cbaenziger) [inline cookbooks/bcpc-hadoop/recipes/oozie.rb]: Why hard code the version strings here? Shouldn't all releases as far as we know need this until they don't?

Comment (aespinosa) [reply cookbooks/bcpc-hadoop/recipes/oozie.rb]: originally i thought it will just be for one release

Comment (aespinosa) [reply cookbooks/bcpc-hadoop/recipes/oozie.rb]: fixed

**Pull request merged**